corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -167,16 +174,18 @@ <h2>{% trans "Manage Organization Structure" %}</h2>
             name="location_search"
             style="max-width:80%;"></select>
 
+            <div style="display:inline; cursor:pointer;" data-bind="if: selected_location_id(), click: clearLocationSelection">
+            <button type="button" class="btn btn-secondary"><i class="fa fa-times" aria-hidden="true"></i></button>
+            </div>
```

Comment:
What does wrapping this in a div give us—was it necessary for formatting/layout, or was it just a sort of dummy element to put the `data-bind` stuff on?

I think (if this isn't ruled out for some other reason I'm missing) my preference would be to put the `data-bind` on the `button` and do away with the wrapping `<div>` tag. You will have to use `visible: ` (which uses css to hide the element it's defined on) rather than `if: ` (which restructures the DOM to remove all nodes _under_ the element it's defined on), since you'll be wanting to hide the element it's being used on (now `<button>`) rather than the elements under it.